bundles/org.openhab.binding.mail/src/main/java/org/openhab/binding/mail/internal/POP3IMAPHandler.java

```diff
@@ -71,7 +71,7 @@ public void initialize() {
         protocol = baseProtocol;
 
         if (config.security == ServerSecurity.SSL) {
-            protocol.concat("s");
+            protocol = protocol.concat("s");
```

Comment:
This is the only useful change here. Please remove all the log-bloating debugging.

Reply:
Not going to happen. Debug logging is needed and useful.

Reply:
As a Codeowner I prefer to keep the badly written style we had before.